src/sentry/models/activity.py

```diff
@@ -95,9 +94,6 @@ def save(self, *args, **kwargs):
         if self.type == Activity.NOTE:
             self.group.update(num_comments=F('num_comments') + 1)
 
-            if self.event:
-                self.event.update(num_comments=F('num_comments') + 1)
```

Comment:
Just to clarify, because this behavior is changing:

Before, our Comments tab was a combination of Activity + Comments for the count.

Now, it'll just be Comments. Which, is accurate to me. But we conflate the meaning of "Comments" tab anyways, which also shows activity events. Now we'll be losing that extra count for activity that happened on an event.

If we care about this, can we just lax the ForeignKey constraint and let the column be simply an integer?